clamurai/Assets/Scripts/Player.cs

```diff
@@ -7,8 +7,8 @@ public class Player : MonoBehaviour
 {
     public const float RUN_SPEED = 10f;
     public const float JUMP_SPEED = 10f;
-    public const float DIST_GROUND = .55f;
-    public const float DIST_SIDE = .5f;
+    public const float DIST_GROUND = .06f;
+    public const float DIST_SIDE = .06f;
```

Comment:
This number needs to match half the width of our player character's terrain collision box (in unity units) otherwise our character won't detect they're grounded when they're partially off of a ledge:

![image](https://github.com/SamBumgardner/clamurai/assets/14338773/2c31f563-c616-4c1c-bb66-977fe75aa09e)

The character's in a falling state here even though they're on the ground because both ground check raycasts are over open air.

Reply:
Other option would be to make the player box collider correspondingly small, but I don't think we want to trim the width this much.

Reply:
Oh I'm sorry! I see what I did. The sides should still be at .5

The reason for the .05 is the aseprite import centers sprites, and I think the previous square had it a the top left corner. That's what caused the entire collider to appear below the sprite

Reply:
ah, that makes sense to me 👍